styles/styles.css

```
@@ -384,23 +580,29 @@ a {
   align-items: center;
   padding: 0;
   gap: 40px;
-  width: 1200px;
+  height: 120px;
   margin: 0 auto;
 }
 
 .footer-contacts-address-phone {
+  text-decoration: none;
   font-size: 40px;
   line-height: 40px;
-  text-align: center;
+  align-items: center;
   color: #000000;
 }
 
 .footer-social-list {
-  width: 150px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 142px;
   margin-right: 200px;
```

```suggestion
  display: flex;
  align-items: center;
```
пока это все не нужно, позже добавим если понадобится